core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -80,6 +80,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('contao.security.two_factor.enforce_backend', $config['security']['two_factor']['enforce_backend']);
         $container->setParameter('contao.localconfig', $config['localconfig'] ?? []);
         $container->setParameter('contao.backend', $config['backend']);
+        $container->setParameter('contao.backend.route_prefix', $config['backend']['route_prefix']);
```

Comment:
Yep, the parameter `contao.backend.route_prefix` is required. There is no normalization process in the extension that "unpacks" the `contao.backend` parameter, as one could think.

<img width="715" alt="Screenshot 2021-11-05 at 19 50 06" src="https://user-images.githubusercontent.com/1284725/140563271-1d3d5126-d0cb-4ff6-920a-350720ad783e.png">

Closing this.

Reply:
Unfortunately, this problem will not disappear by resolving the conversation. It shows us that the changes from #1999 are incorrect, because they take the entire `contao.backend` namespace instead of separate sub-namespaces. We will have to fix this first.

Reply:
See https://github.com/contao/contao/pull/3653

Reply:
To be honest, I don't see a problem or a necessity for #3653 since the `contao.backend` "namespace" is not blocked.
You were able to fetch the route prefix by `$this->container->get('contao.backend')['route_prefix']` or `$this->container->get('contao.backend.route_prefix')`. The latter one was only required because it is used directly as a service parameter.